node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -147,12 +144,10 @@ impl HighwayConfig {
         HighwayConfig {
             era_duration: TimeDiff::from(rng.gen_range(600_000, 604_800_000)),
             minimum_era_height: rng.gen_range(5, 100),
-            booking_duration: TimeDiff::from(rng.gen_range(600_000, 864_000_000)),
-            entropy_duration: TimeDiff::from(rng.gen_range(600_000, 10_800_000)),
-            voting_period_duration: TimeDiff::from(rng.gen_range(600_000, 172_800_000)),
-            finality_threshold_percent: rng.gen_range(0, 101),
+            finality_threshold_fraction: Ratio::new(rng.gen_range(0, 101), 100),
```

Comment:
Let's exclude 0 and 100 here.

Reply:
Actually, it would be good in general if we could reject chainspecs where this is not in (0, 1), and where `reduced_reward_multiplier` is not in [0, 1].

Reply:
Done now.